.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -6,58 +6,48 @@ body:
   attributes:
     value: |
        :pray: Thanks for taking the time to fill out this bug report! Feel free to ping us on [Wing Slack](https://t.winglang.io/slack) if you have any questions or need help.
+
 - type: markdown
   attributes:
     value: |
        ## Bug Report
+
 - type: textarea
   id: i-tried-this
   attributes:
     label: "I tried this:"
     placeholder: "What did you try to do? A code snippet or exact command line is ideal."
   validations:
     required: true
+
 - type: textarea
   id: instead-what-happened
   attributes:
     label: "This happened:"
-    placeholder: "What happend instead of what you've expected?"
+    placeholder: "What happened instead of what you've expected?"
   validations:
     required: true
+
 - type: textarea
```

Comment:
I think we can make this optional. In many cases, a simple error message in the "actual" section is sufficient.